song_match/effect/effects/game_won.py

```diff
@@ -0,0 +1,26 @@
+from cozmo.anim import Triggers
+
+from song_match.effect.effect import Effect
+
+class GameWonEffect(Effect):
+
+    async def play(self, is_player: bool = True) -> None:
+        """Play the correct sequence effect.
```

Comment:
You should update `Play the correct sequence effect.` to something like `Play the game over effect.`.